docs/CONTRIBUTING.md

```diff
@@ -1,5 +1,16 @@
 # Contribution Guidelines
 
+## Contents
+
+- [Setup developer environment](#setup-developer-environment)
+- [Build the code](#build-the-code)
+- [Build with docker](#build-with-docker)
+- [Update assets](#update-assets)
+- [Authoring PRs](#authoring-prs)
+    - [Commit Format](#commit-format)
```

Comment:
nit: this has two extra space of indentation.

Reply:
Fixing it